packages/scenario-harness/ui/app.js

```
@@ -761,18 +762,28 @@ function readSessionStartSnapshot() {
   };
 }
 
+// Push the composed snapshot first so the scenario's resolver sees it, then
+// publish the trigger event. `/api/session-start/snapshot` awaits a 204
+// before we publish, so there's no push-vs-fire race.
+async function fireSessionStart(data, label) {
+  try {
+    await post("/api/session-start/snapshot", readSessionStartSnapshot());
+    await post("/api/bus/publish", { event: "session.changed", data });
+  } catch (e) {
+    alert(`${label} failed: ${e.message}`);
```

ESLint

[error] 'alert' is not defined.

(no-undef)